common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -889,6 +889,19 @@ def user_details_force_sync(auth_entry, strategy, details, user=None, *args, **k
                 changed[provider_field] = current_value
                 setattr(model, field, provider_value)
 
+        # Generate fullname only for IES IDP.
```

Comment:
```suggestion
        """Generate full name only for IES IDP.
        We deliberately left these values hard-coded instead of using Django settings because
        it would force us to add custom settings to the edx platform code,
        which we try to avoid as we might lose track of that kind of setting."""
```

Reply:
What do you think of putting this info as inline comments? or as a docstring but below the function definition

Reply:
I'm fine with inline comments, @alexjmpb